plugins/woocommerce-admin/client/marketing/overview-multichannel/Channels/RegisteredChannelCardBody.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import GridiconCheckmarkCircle from 'gridicons/dist/checkmark-circle';
+import GridiconSync from 'gridicons/dist/sync';
+import GridiconNotice from 'gridicons/dist/notice';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { PluginCardBody } from '~/marketing/components';
+import { RegisteredChannel, SyncStatusType } from '~/marketing/types';
+import './RegisteredChannelCardBody.scss';
+
+type RegisteredChannelCardBodyProps = {
+	registeredChannel: RegisteredChannel;
+};
+
+type SyncStatusPropsType = {
+	status: SyncStatusType;
+};
+
+const iconSize = 18;
+const className = 'woocommerce-marketing-sync-status';
```

Comment:
💅  This constants I would use UPPERCASE format.

Reply:
Since it's defined at the top file, I take as local constants and hence I think it will be most correct UPPERCASE. Also one of them now s even exportable.